CHANGELOG.md

```diff
@@ -3,6 +3,95 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 6.0.0
+
+_Mar 3, 2023_
+
+We'd like to offer a big thanks to the 12 contributors who made this release possible. Here are some highlights ✨:
+
+- 🎁 The row pinning is no longer experimental. (#8055) @MBilalShafi
+
+  You can now use the row pinning without the `experimentalFeatures.rowPinning` flag enabled.
+
+  ```diff
+   <DataGridPro
+  -  experimentalFeatures={{ rowPinning: true }}
+   />
+  ```
+- ⚡️ You can now memoize rows and cells. (#7846) @m4theushw
+- ✨ Fields have a distinct visual state when empty (#8069) @LukasTy
+- 🌍 Import Czech (cs-CZ) locale (#8113) @BlastyCZ
+- 🌍 Improve Arabic (ar-SD) locale (#8100) @atf98
+
+### `@mui/x-data-grid@v6.0.0` / `@mui/x-data-grid-pro@v6.0.0` / `@mui/x-data-grid-premium@v6.0.0`
+
+#### Breaking changes
+
+- Add typing for `componentsProps` and `slotProps` props
```

Comment:
Should we list all the changes from alpha an beta in the stable release notes?

Reply:
I'd question changelog readability at that point. 🙈 
It would be pretty massive.
The linked `whats-new` page has links to package migration pages with all the BCs listed...
Maybe that's enough?

We could maybe add [a link to all BC PRs](https://github.com/mui/mui-x/pulls?page=1&q=is%3Apr+label%3A%22breaking+change%22+label%3Av6.x+is%3Aclosed) instead. 🤔

Reply:
Maybe we can list all changes only on the GitHub release page?
My point here is that it could be useful for people who aren't familiar with the changes we've done in alpha and beta releases.
Take next.js for example - they release a lot of `canary` versions, but the major release list all the changes since the previous stable release https://github.com/vercel/next.js/releases/tag/v13.0.0

Reply:
If we do, then we need to clean the list because there is a lot of intermediary changes that make no sense (all the fixes on the field for example, they just did not exist before).

Reply:
For the pickers, I think the migration guide is a good list of everything that changed, better than a changelog because since we replaced the old pickers with the new one, the list never has the changes in behavior themselves.

Reply:
> Take next.js for example - they release a lot of `canary` versions, but the major release list all the changes since the previous stable release

That's a good example, but they have way fewer BCs (just 5 in v13).
With our codebase maturity and BC count level, I think that re-listing all of them is less feasible.

I do agree that it would be useful for someone to scan through major release changes and note the possible topics in need of attention, but that would also be a bit of a chore on our side to create a compact, yet useful list of all BCs.
And given the fact that the migration guide has a quite well-formatted list, I doubt the ROI on this effort is worth it. 🙈

Reply:
> The linked whats-new page has links to package migration pages with all the BCs listed

Perhaps it's a good idea to add the links to the migration guides directly on the changelog.